test/wasm/rlexe.xml

```diff
@@ -49,4 +49,11 @@
        <compile-flags>-on:Wasm  -on:wasmlazytrap</compile-flags>
     </default>
 </test>
+<test>
+    <default>
+       <files>i32_unreachable.js</files>
```

Comment:
why name it `i32_unreachable` ? The fact that you use int32 is irrelevant to the test. Just call the file `unreachable.js`